shared/javascripts/tooltip.js

```diff
@@ -1,7 +1,7 @@
 /**
  * @fileOverview This script defines a tooltip to indicate that the content is
  * not a natural element of the page.
- * 
+ *
```

Comment:
I generally like having whitespace cleanup commits separate, but it is no big deal in this case.